File: 백준/Bronze/19944. 뉴비의 기준은 뭘까？/뉴비의 기준은 뭘까？.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String[] str = sc.nextLine().split(" ");
        int n = Integer.parseInt(str[0]);
        int m = Integer.parseInt(str[1]);

        if (m == 1 || m == 2)
            System.out.println("NEWBIE!");
        else if (m <= n)
            System.out.println("OLDBIE!");
        else
            System.out.println("TLE!");
    }
}

File: 백준/Bronze/19944. 뉴비의 기준은 뭘까？/뉴비의 기준은 뭘까？.cs
string[] str = Console.ReadLine().Split();

int n = int.Parse(str[0]);
int m = int.Parse(str[1]);

if (m == 1 || m == 2)
    Console.WriteLine("NEWBIE!");
else if (m <= n)
    Console.WriteLine("OLDBIE!");
else
    Console.WriteLine("TLE!");